app/views/provider_management/external_users/enable_confirmation.html.haml

```diff
@@ -0,0 +1,14 @@
+.govuk-grid-row
+  .govuk-grid-column-two-thirds
+    = content_for :page_title, flush: true do
+      = t('.page_title', external_user: @external_user.name)
```

Comment:
Nit: this content block will not be printed within the body tag, therefore does not need to be wrapped within the grid